src/com/sun/ts/tests/signaturetest/saaj/SAAJSigTest.java

```diff
@@ -123,14 +123,9 @@ public void setup(String[] args, Properties p) throws Fault {
    * define which sig driver we will use
    */
   protected SignatureTestDriver getSigTestDriver() {
-    String version = (String) System.getProperties().get("java.version");
-    if (version.startsWith("9"))
-      driver = new Jdk9SigTestDriver();
-    if (driver == null) {
-      driver = SignatureTestDriverFactory
-          .getInstance(SignatureTestDriverFactory.SIG_TEST);
-    }
-
+    driver = SignatureTestDriverFactory
```

Comment:
SAAJ standalone TCK Sigtest update required?

Reply:
The SAAJ TCK was using Jdk9SigTestDriver which is not required and removed.